### PR TITLE
docs(Advanced guides): Translate Portals

### PR DESCRIPTION
**Pull request by henry61024**
Translate Advanced guides - `Portals`

### CODE REVIEW

**Review by neighborhood999 — COMMENTED** [review]
Thanks for helping to translate! I left some suggestions and changes, feel free to let me know if you have any questions.

File: content/docs/portals.md [hunk 1]
```diff
@@ -27,34 +27,34 @@ render() {
...
 ```
 
-A typical use case for portals is when a parent component has an `overflow: hidden` or `z-index` style, but you need the child to visually "break out" of its container. For example, dialogs, hovercards, and tooltips.
+一個典型的 portal 使用案例是，當 parent component 有 `overflow: hidden` 或者 `z-index` 的樣式時，卻仍需要 child 在視覺上「跳出」其容器的狀況。例如 dialog，hovercard 與 tooltip 都屬於此案例。
 
-> Note:
+> 注意：
 >
-> When working with portals, remember that [managing keyboard focus](/docs/accessibility.html#programmatically-managing-focus) becomes very important.
+> 當使用 portal 時，需記得[控管鍵盤 focus](/docs/accessibility.html#programmatically-managing-focus) 會變得非常重要。
```
[the remark this answers is not in the document]
Comment (henry61024) [reply content/docs/portals.md]: Fixed in ebd1a3b

But I removed the `，` before `對於無障礙功能會變得非常重要`

Please see if the change is reasonable for you :)

File: content/docs/portals.md [hunk 2]
```diff
@@ -27,34 +27,34 @@ render() {
...
+  // `domNode` 可以是任何在隨意位置的合法 DOM node。
   return ReactDOM.createPortal(
     this.props.children,
     domNode
   );
 }
 ```
 
-A typical use case for portals is when a parent component has an `overflow: hidden` or `z-index` style, but you need the child to visually "break out" of its container. For example, dialogs, hovercards, and tooltips.
+一個典型的 portal 使用案例是，當 parent component 有 `overflow: hidden` 或者 `z-index` 的樣式時，卻仍需要 child 在視覺上「跳出」其容器的狀況。例如 dialog，hovercard 與 tooltip 都屬於此案例。
```
[the remark this answers is not in the document]
Comment (henry61024) [reply content/docs/portals.md]: Fixed in ebd1a3b

File: content/docs/portals.md [hunk 3]
```diff
@@ -27,34 +27,34 @@ render() {
...
 ```
 
-However, sometimes it's useful to insert a child into a different location in the DOM:
+然而在某些狀況下，將 child 插入不同位置的 DOM 內十分好用：
 
 ```js{6}
 render() {
-  // React does *not* create a new div. It renders the children into `domNode`.
-  // `domNode` is any valid DOM node, regardless of its location in the DOM.
+  // React *不會* 創建新的 div。它會將 children render 進 `domNode` 中。
```
[the remark this answers is not in the document]
Comment (henry61024) [reply content/docs/portals.md]: Fixed in ebd1a3b

File: content/docs/portals.md [hunk 4]
```diff
@@ -1,24 +1,24 @@
...
 
-Portals provide a first-class way to render children into a DOM node that exists outside the DOM hierarchy of the parent component.
+Portal 提供一個優秀方法來讓 children 可以 render 到 parent component DOM 樹以外的 DOM 節點。
 
 ```js
 ReactDOM.createPortal(child, container)
 ```
 
-The first argument (`child`) is any [renderable React child](/docs/react-component.html#render), such as an element, string, or fragment. The second argument (`container`) is a DOM element.
+第一個參數 (`child`) 是任何[可 render 的 React child](/docs/react-component.html#render)，例如 element，string 或者 fragment。第二個參數 (`container`) 則是一個 DOM element。
```
[the remark this answers is not in the document]
Comment (henry61024) [reply content/docs/portals.md]: Fixed in ebd1a3b

**Review by neighborhood999 — APPROVED** [review]
Thank you for helping! ❤️

**Pull request merged**